src/Apps/Artist/Routes/Consign/__tests__/ConsignRoute.test.tsx

```diff
@@ -79,7 +79,9 @@ describe("ConsignRoute", () => {
           .find("ArtistConsignHeader")
           .find("RouterLink")
           .html()
-      ).toContain(`href="/consign/submission"`)
+      ).toContain(
+        `href="/consign/submission?contextPath=/artist/alex-katz/consign&amp;subject=Request a price estimate"`
```

Comment:
```suggestion
        `href="/consign/submission?contextPath=/artist/alex-katz/consign&subject=Request+a+price+estimate"`
```

I might have misspoken in our DMs, but I would expect the contents of the `contextPath` and `subject` params individually to be encoded.

Another example: If a test passed `contextPath="/some/path?someArg=someValue&otherArg=otherValue"`, I'd expect `href` to equal `/consign/submission?contextPath=%2Fconsign%2Fsubmission%3FsomeArg%3DsomeValue%26otherArg%3DotherValue&subject=Request+a+price+estimate

Reply:
Ahh, giving this a shot...

Reply:
How does [this commit](https://github.com/artsy/reaction/pull/3419/commits/6bbf6d081a97127ba496b37a56b980ccba6aeabd) look? Using `encodeURIComponent` inserted a `%20` rather than a +.

Reply:
👍 Looks good, thanks for digging into that!

Reply:
👍 this query param stuff is honestly confusing lol